pkgs/applications/misc/osmium-tool/default.nix

```
@@ -48,6 +48,7 @@ stdenv.mkDerivation rec {
   meta = with lib; {
     description = "Multipurpose command line tool for working with OpenStreetMap data based on the Osmium library";
     homepage = "https://osmcode.org/osmium-tool/";
+    changelog = "https://github.com/osmcode/osmium-tool/blob/${version}/CHANGELOG.md";
```

```suggestion
    changelog = "https://github.com/osmcode/osmium-tool/blob/v${version}/CHANGELOG.md";
```

https://github.com/osmcode/osmium-tool/blob/v1.13.1/CHANGELOG.md